docs/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
```

Comment:
Maybe adding something more meaningful for SEO, such as:

`<title>RapiDAST - Dynamic Application Security Testing</title>
`